confdb/aclocal_libs.m4

```diff
@@ -127,6 +127,34 @@ AC_DEFUN([PAC_CHECK_HEADER_LIB_OPTIONAL],[
     fi
 ])
 
+dnl PAC_CHECK_HEADER_LIB_EXTRAFLAGS_OPTIONAL(with_option, header.h, libname, function, other_lib, extra_CPPFLAGS)
+dnl Check optional library. The results are in $pac_have_$1.
+AC_DEFUN([PAC_CHECK_HEADER_LIB_EXTRAFLAGS_OPTIONAL],[
```

Comment:
You don't need a new macro to just add an optional argument. You can use `m4_if`, for example -
```
m4_if($6, [], [], [
     dnl $6 is not empty
     do your optional behavior
])
```